source/fe/fe_simplex_p.cc

```diff
@@ -323,13 +326,14 @@ template <int dim, int spacedim>
 FE_SimplexPoly<dim, spacedim>::FE_SimplexPoly(
   const BarycentricPolynomials<dim>              polynomials,
   const FiniteElementData<dim>                  &fe_data,
+  bool                                           prolongation_is_additive,
   const std::vector<Point<dim>>                 &unit_support_points,
   const std::vector<std::vector<Point<dim - 1>>> unit_face_support_points,
   const FullMatrix<double>                      &interface_constraints)
   : dealii::FE_Poly<dim, spacedim>(
       polynomials,
       fe_data,
-      std::vector<bool>(fe_data.dofs_per_cell),
+      std::vector<bool>(fe_data.dofs_per_cell, prolongation_is_additive),
```

Comment:
Note that these changes follow the behavior of `FE_Q` and `FE_DGQ`, respectively: The DG element is made additive by a true projection, whereas the continuous element is simply injecting values. As a result, the matrices need to be determined in different ways, and only the DG case can use the `FETools` setup.